robohive/utils/prompt_utils.py

```diff
@@ -18,12 +18,12 @@
 # Define verbosity levels
 class Prompt(enum.IntEnum):
     """Prompt verbosity types"""
-    NONE = 0    #  default (lowest priority)
+    ALL = 0     # print everything (lowest priority)
```

Comment:
shouldn't be an option to print `NONE` ?

Reply:
Great point. Added an option.